app/Http/Controllers/Wiki/LanguagesController.php

```diff
@@ -203,27 +203,32 @@ public function search($search_text)
             $GLOBALS['i18n_id'],
             $key
         ];
-        $languages = cacheRemember('languages_search', $cache_params, now()->addDay(), function () use ($formatted_search, $limit, $key) {
-            $languages = Language::includeAutonymTranslation()
-                ->includeCurrentTranslation()
-                ->filterableByNameOrAutonym($formatted_search)
-                ->isContentAvailable($key)
-                ->select([
-                    'languages.id',
-                    'languages.glotto_id',
-                    'languages.iso',
-                    'languages.name as backup_name',
-                    'current_translation.name as name',
-                    'autonym.name as autonym',
-                ]);
-            $languages = $languages->paginate($limit);
-            $languages_return = fractal(
-                $languages->getCollection(),
-                LanguageTransformer::class,
-                $this->serializer
-            );
-            return $languages_return->paginateWith(new IlluminatePaginatorAdapter($languages));
-        });
+        $languages = cacheRemember(
+            'languages_search',
+            $cache_params,
+            now()->addDay(),
+            function () use ($formatted_search, $limit, $key) {
+                $languages = Language::includeAutonymTranslation()
+                    ->includeCurrentTranslation()
+                    ->filterableByNameAndKey($formatted_search, $key)
```

Comment:
This is the replacement for the code removed above, but filterableByNameAndKey doesn't include the antonym clause.